release/mk-langpack.sh

```diff
@@ -8,12 +8,20 @@
 #  To force a build with todays snapshot pass "--force" as the first parameter
 #
 
+test -e ~/.server.config && . ~/.server.config || exit 1
+
 # Location where the langpack files are published
-webpath=`cat ~/.website-path`
-filedir=${webpath}/content/Versions/langpack
+filedir=$SQUID_WWW_PATH/content/Versions/langpack
+
+# auto-detect the latest 'major' version number
+SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
-# Location of the repository checkout
-vcsdir=/srv/sources/squid-master
+# Location of the latest versions repository checkout
+vcsdir="${SQUID_VCS_PATH}squid-${SQUID_RELEASE}"
```

Comment:
This line contradicts `SQUID_VCS_PATH` definition (and the provided example) which places versions in subdirectories:

```suggestion
vcsdir="${SQUID_VCS_PATH}/squid-${SQUID_RELEASE}"
```